wire-ios-request-strategy/Sources/Request Strategies/Feature configurations/FeatureConfigRequestStrategy.swift

```diff
@@ -45,14 +54,37 @@ public final class FeatureConfigRequestStrategy: AbstractRequestStrategy {
         configuration = [
             .allowsRequestsWhileOnline,
             .allowsRequestsDuringQuickSync,
+            .allowsRequestsDuringSlowSync,
             .allowsRequestsWhileWaitingForWebsocket,
             .allowsRequestsWhileInBackground
         ]
     }
 
+    deinit {
+        task?.cancel()
+    }
+
     // MARK: - Request
 
     public override func nextRequestIfAllowed(for apiVersion: APIVersion) -> ZMTransportRequest? {
+        if isSlowSyncing, task == nil {
+            task = Task { [weak self] in
+                guard let self, !Task.isCancelled else { return }
+
+                WireLogger.conversation.info("FeatureConfigRequestStrategy: slow sync start fetch feature config!")
+
+                if #available(iOS 16, *) {
+                    try? await Task.sleep(for: .seconds(1))
+                } else {
+                    try? await Task.sleep(nanoseconds: 1_000_000)
+                }
```

Comment:
why do we sleep here?

Reply:
This is still a draft PR and the implementation is not done yet 😅